app/Models/Language/Language.php

```diff
@@ -451,7 +459,7 @@ public function scopeFilterableByNameAndKey($query, $name, $key)
                     join ' . $dbp_prod . '.bibles b on bfc.bible_id = b.id
                     WHERE uk.key = ? and lang_trans.language_source_id = b.language_id
                 )',
-            );
+            )->where('lang_trans.priority', '=', $priority_union);
```

Comment:
it must filter the language_translations by  highest priority records to avoid to search records with low and high priority.